diff -up qt-everywhere-opensource-src-5.2.0/qttools/src/src.pro.orig qt-everywhere-opensource-src-5.2.0/qttools/src/src.pro
--- qt-everywhere-opensource-src-5.2.0/qttools/src/src.pro.orig	2014-01-25 12:59:16.425031815 +0100
+++ qt-everywhere-opensource-src-5.2.0/qttools/src/src.pro	2014-01-25 13:00:27.952012879 +0100
@@ -22,7 +22,7 @@ mac {
 }
 
 android {
-    SUBDIRS += androiddeployqt
+    SUBDIRS = androiddeployqt
 }
 
 qtHaveModule(dbus): SUBDIRS += qdbus
