diff -up Qt5/qttools/src/linguist/linguist.pro.orig Qt5/qttools/src/linguist/linguist.pro
--- Qt5/qttools/src/linguist/linguist.pro.orig	2014-02-23 14:29:11.583916184 +0100
+++ Qt5/qttools/src/linguist/linguist.pro	2014-02-23 14:33:55.779701433 +0100
@@ -4,6 +4,7 @@ SUBDIRS  = \
     lupdate \
     lconvert
 !no-png:qtHaveModule(widgets): SUBDIRS += linguist
+android: SUBDIRS -= linguist
 
 qtNomakeTools( \
     linguist \
diff -up Qt5/qttools/src/src.pro.orig Qt5/qttools/src/src.pro
--- Qt5/qttools/src/src.pro.orig	2014-02-23 14:15:34.304036831 +0100
+++ Qt5/qttools/src/src.pro	2014-02-23 14:32:40.548817140 +0100
@@ -23,6 +23,8 @@ mac {
 
 android {
     SUBDIRS += androiddeployqt
+    SUBDIRS -= assistant pixeltool designer
+    linguist.depends -= designer
 }
 
 qtHaveModule(dbus): SUBDIRS += qdbus
