diff -up qt-everywhere-opensource-src-5.2.0/qt.pro.orig qt-everywhere-opensource-src-5.2.0/qt.pro
--- qt-everywhere-opensource-src-5.2.0/qt.pro.orig	2014-01-25 12:58:45.173477010 +0100
+++ qt-everywhere-opensource-src-5.2.0/qt.pro	2014-01-25 13:00:03.836356417 +0100
@@ -68,7 +68,7 @@ addModule(qtactiveqt, qtbase)
 addModule(qt3d, qtdeclarative)
 addModule(qtjsondb, qtdeclarative)
 addModule(qtsystems, qtbase, qtdeclarative)
-addModule(qtlocation, qtbase, qt3d qtsystems qtmultimedia)
+addModule(qtlocation, qtbase, qt3d qtsystems qtmultimedia qtdeclarative)
 addModule(qtsensors, qtbase, qtdeclarative)
 addModule(qtconnectivity, qtbase, qtdeclarative)
 addModule(qtfeedback, qtdeclarative, qtmultimedia)
