diff -up android-ndk-r9c/build/tools/build-gnu-libstdc++.sh.orig android-ndk-r9c/build/tools/build-gnu-libstdc++.sh
--- android-ndk-r9c/build/tools/build-gnu-libstdc++.sh.orig	2014-02-16 20:32:24.008371860 +0100
+++ android-ndk-r9c/build/tools/build-gnu-libstdc++.sh	2014-03-26 20:03:11.363469785 +0100
@@ -19,6 +19,9 @@
 #  that contains valid plaforms files and toolchain binaries.
 #
 
+# CHANGE NOTICE: THIS FILE HAS BEEN MODIFIED BY OPENORIENTEERING.
+#                http://www.openorienteering.org
+
 # include common function and variable definitions
 . `dirname $0`/prebuilt-common.sh
 
@@ -71,6 +74,7 @@ WITH_DEBUGGING_INFO=
 register_var_option "--with-debugging-info" WITH_DEBUGGING_INFO "Build with -g.  STL is still built with optimization but with debugging info"
 
 register_jobs_option
+register_try64_option
 
 extract_parameters "$@"
 
