diff -urwN -x toolchains mingw-builds.orig/README.OpenOrienteering.txt mingw-builds/README.OpenOrienteering.txt
--- mingw-builds.orig/README.OpenOrienteering.txt	1970-01-01 00:00:00 +0000
+++ mingw-builds/README.OpenOrienteering.txt	2013-08-19 03:13:04 +0000
@@ -0,0 +1,18 @@
+This directory contains the mingw-builds [1] scripts patched by
+OpenOrienteering [2]. Modifications to mingw-builds were made in order to
+- closely reproduce a particular MinGW binary archive which is distributed
+  by MinGW builds and/or the Qt project® [3],
+- reduce build issues triggered by the MSYS environment,
+- speed up the compilation of GCC,
+- update license documents in the created binary archive,
+- mark the created GCC binaries as being built by OpenOrienteering, and
+  consequently give OpenOrienteering's support page as bug URL.
+
+All modification are marked by "# Modification from OpenOrienteering".
+
+[1] http://sourceforge.net/projects/mingwbuilds/
+[2] http://sourceforge.net/projects/oorienteering/
+[3] http://qt-project.org/
+
+Qt is a registered trade mark of Digia plc and/or its subsidiaries.
+
diff -urwN -x toolchains mingw-builds.orig/build mingw-builds/build
--- mingw-builds.orig/build	2013-05-22 09:53:15 +0000
+++ mingw-builds/build	2013-08-22 22:17:26 +0000
@@ -60,8 +60,8 @@
 # **************************************************************************
 
 MINGW_BUILDS_VERSION="MinGW-builds-2.0.0"
-PKG_VERSION="Built by MinGW-builds project"
-BUG_URL=http://sourceforge.net/projects/mingwbuilds/
+PKG_VERSION="built using MinGW-builds by the OpenOrienteering project" # Modification from OpenOrienteering
+BUG_URL=http://sourceforge.net/projects/oorienteering/support/         # Modification from OpenOrienteering
 
 x32_HOST_MINGW_PATH_URL=http://sourceforge.net/projects/mingwbuilds/files\
 /host-windows/releases/4.7.2/32-bit/threads-posix/sjlj/x32-4.7.2-release-posix-sjlj-rev11.7z
@@ -135,6 +135,7 @@
 [[ $SHOW_LOG_ON_ERROR == yes && ! -f $LOGVIEWER ]] && {
 	LOGVIEWER=/c/Windows/system32/notepad.exe
 }
+LOGVIEWER="echo Log file:" # Modification by OpenOrienteering
 
 [[ -d /mingw ]] && {
 	die "please remove \"/mingw\" directory. terminate."
@@ -467,7 +469,7 @@
 	gdbinit
 	gdb
 	gdb-wrapper
-	make_cvs
+	make_git
 	cleanup
 	licenses
 	put-versions
diff -urwN -x toolchains mingw-builds.orig/licenses/mingw-w64/COPYING mingw-builds/licenses/mingw-w64/COPYING
--- mingw-builds.orig/licenses/mingw-w64/COPYING	2013-05-22 09:52:47 +0000
+++ mingw-builds/licenses/mingw-w64/COPYING	2013-08-19 03:13:04 +0000
@@ -2,7 +2,7 @@
 in the Public Domain, BSD, or LGPL this Software is provided under the
 Zope Public License (ZPL) Version 2.1.
 
-Copyright (c) 2009, 2010 by the mingw-w64 project
+Copyright (c) 2009 - 2013 by the mingw-w64 project
 
 See the AUTHORS file for the list of contributors to the mingw-w64 project.
 
diff -urwN -x toolchains mingw-builds.orig/licenses/mingw-w64/COPYING.MinGW-w64-runtime.txt mingw-builds/licenses/mingw-w64/COPYING.MinGW-w64-runtime.txt
--- mingw-builds.orig/licenses/mingw-w64/COPYING.MinGW-w64-runtime.txt	2013-05-22 09:52:47 +0000
+++ mingw-builds/licenses/mingw-w64/COPYING.MinGW-w64-runtime.txt	2013-08-19 03:13:04 +0000
@@ -12,7 +12,7 @@
 Overall copyright notice
 ========================
 
-Copyright (c) 2009, 2010 by the mingw-w64 project
+Copyright (c) 2009, 2010, 2011, 2012, 2013 by the mingw-w64 project
 
 This license has been certified as open source. It has also been designated
 as GPL compatible by the Free Software Foundation (FSF).
diff -urwN -x toolchains mingw-builds.orig/licenses/mingw-w64/DISCLAIMER.PD mingw-builds/licenses/mingw-w64/DISCLAIMER.PD
--- mingw-builds.orig/licenses/mingw-w64/DISCLAIMER.PD	2013-05-22 09:52:47 +0000
+++ mingw-builds/licenses/mingw-w64/DISCLAIMER.PD	2013-08-19 03:13:04 +0000
@@ -1,9 +1,9 @@
 /**
  * DISCLAIMER
  * This file has no copyright assigned and is placed in the Public Domain.
- * This file is a part of the w64 mingw-runtime package.
+ * This file is part of the mingw-w64 runtime package.
  *
- * The w64 mingw-runtime package and its code is distributed in the hope that it 
+ * The mingw-w64 runtime package and its code is distributed in the hope that it 
  * will be useful but WITHOUT ANY WARRANTY.  ALL WARRANTIES, EXPRESSED OR 
  * IMPLIED ARE HEREBY DISCLAIMED.  This includes but is not limited to 
  * warranties of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
diff -urwN -x toolchains mingw-builds.orig/scripts/binutils.sh mingw-builds/scripts/binutils.sh
--- mingw-builds.orig/scripts/binutils.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/binutils.sh	2013-08-19 03:13:04 +0000
@@ -106,4 +106,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/bzip2.sh mingw-builds/scripts/bzip2.sh
--- mingw-builds.orig/scripts/bzip2.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/bzip2.sh	2013-08-19 03:13:04 +0000
@@ -54,7 +54,8 @@
 #
 
 EXECUTE_AFTER_PATCH=(
-	"autogen.sh"
+	"./autogen.sh"       # Modification from OpenOrienteering
+	"test -f config.sub" # Modification from OpenOrienteering
 )
 
 #
@@ -88,4 +89,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/cleanup.sh mingw-builds/scripts/cleanup.sh
--- mingw-builds.orig/scripts/cleanup.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/cleanup.sh	2013-08-19 03:13:04 +0000
@@ -41,6 +41,9 @@
 	rm -rf $PREFIX/{mingw,share/info,share/man}
 
-	touch $BUILDS_DIR/cleanup.marker
+#	touch $BUILDS_DIR/cleanup.marker # Modification from OpenOrienteering
 }
 
+# Modification from OpenOrienteering
+find $SRCS_DIR -name "*.stackdump" -o -name "exec-after_install-*" -exec rm -f {} \;
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/cloog.sh mingw-builds/scripts/cloog.sh
--- mingw-builds.orig/scripts/cloog.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/cloog.sh	2013-08-19 03:13:05 +0000
@@ -82,4 +82,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp LICENSE* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/expat.sh mingw-builds/scripts/expat.sh
--- mingw-builds.orig/scripts/expat.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/expat.sh	2013-08-19 03:13:05 +0000
@@ -77,4 +77,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/functions.sh mingw-builds/scripts/functions.sh
--- mingw-builds.orig/scripts/functions.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/functions.sh	2013-08-19 03:13:05 +0000
@@ -412,7 +412,7 @@
 				}
 				[[ $_last =~ .exe ]] && {
 					printf "%-50s" "--> execute $arch_it: \"$_last\" ... "
-					( cd $3/$arch_it && $_last > $_run_log 2>&1 )
+					( cd $3/$arch_it && ./$_last > $_run_log 2>&1 ) # Modification from OpenOrienteering
 					_result=$?
 					[[ $_result == 0 ]] && {
 						echo "-> $_result -> done"
diff -urwN -x toolchains mingw-builds.orig/scripts/gcc-4.8.0.sh mingw-builds/scripts/gcc-4.8.0.sh
--- mingw-builds.orig/scripts/gcc-4.8.0.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/gcc-4.8.0.sh	2013-08-19 03:13:05 +0000
@@ -123,7 +123,7 @@
 #
 
 MAKE_FLAGS=(
-	-j1
+	-j$JOBS # Modification from OpenOrienteering
 	all
 )
 
@@ -135,4 +135,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/gdb.sh mingw-builds/scripts/gdb.sh
--- mingw-builds.orig/scripts/gdb.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/gdb.sh	2013-08-19 03:13:05 +0000
@@ -90,4 +90,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/gmp.sh mingw-builds/scripts/gmp.sh
--- mingw-builds.orig/scripts/gmp.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/gmp.sh	2013-08-19 03:13:05 +0000
@@ -79,4 +79,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/isl.sh mingw-builds/scripts/isl.sh
--- mingw-builds.orig/scripts/isl.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/isl.sh	2013-08-19 03:13:05 +0000
@@ -38,7 +38,7 @@
 VERSION=0.11.1
 NAME=$ARCHITECTURE-isl-${VERSION}-$LINK_TYPE_SUFFIX
 SRC_DIR_NAME=isl-${VERSION}
-URL=ftp://ftp.linux.student.kuleuven.be/pub/people/skimo/isl//isl-${VERSION}.tar.bz2
+URL=ftp://gcc.gnu.org/pub/gcc/infrastructure/isl-${VERSION}.tar.bz2 # Modification from OpenOrienteering
 TYPE=.tar.bz2
 PRIORITY=prereq
 
@@ -80,4 +80,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp LICENSE* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/libffi.sh mingw-builds/scripts/libffi.sh
--- mingw-builds.orig/scripts/libffi.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/libffi.sh	2013-08-19 03:13:05 +0000
@@ -77,4 +77,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp LICENSE* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/libgnurx.sh mingw-builds/scripts/libgnurx.sh
--- mingw-builds.orig/scripts/libgnurx.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/libgnurx.sh	2013-08-19 03:13:05 +0000
@@ -93,6 +93,8 @@
 
 EXECUTE_AFTER_INSTALL=(
 	"cp -f $LIBS_DIR/lib/libgnurx.a $LIBS_DIR/lib/libregex.a"
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"     # Modification from OpenOrienteering
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/" # Modification from OpenOrienteering
 )
 
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/libiconv-x32.sh mingw-builds/scripts/libiconv-x32.sh
--- mingw-builds.orig/scripts/libiconv-x32.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/libiconv-x32.sh	2013-08-19 03:13:05 +0000
@@ -77,4 +77,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/licenses.sh mingw-builds/scripts/licenses.sh
--- mingw-builds.orig/scripts/licenses.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/licenses.sh	2013-08-19 03:13:05 +0000
@@ -35,10 +35,8 @@
 
 # **************************************************************************
 
-[[ ! -f $BUILDS_DIR/licenses.marker ]] && {
-	cp -rf $TOP_DIR/licenses $PREFIX/
-
-	touch $BUILDS_DIR/licenses.marker
-}
+# Modification from OpenOrienteering
+mkdir -p $PREFIX/licenses
+cp -rf $TOP_DIR/licenses/mingw-w64 $PREFIX/licenses/
 
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/make_git.sh mingw-builds/scripts/make_git.sh
--- mingw-builds.orig/scripts/make_git.sh	2013-05-22 09:53:15 +0000
+++ mingw-builds/scripts/make_git.sh	2013-08-22 22:20:59 +0000
@@ -35,27 +35,34 @@
 
 # **************************************************************************
 
+# Modification from OpenOrienteering
+VERSION=40a49f244da5b417af8bede84ac221cee2318d88 # master @ 2013-07-22
 NAME=make_git
-SRC_DIR_NAME=make_git
-URL="http://git.savannah.gnu.org/cgit/make.git"
-TYPE=git
+SRC_DIR_NAME=${NAME}
+URL=http://git.savannah.gnu.org/cgit/make.git/snapshot/make-${VERSION}.tar.gz
+TYPE=.tar.gz
 REV=
 PRIORITY=extra
 
 #
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_UNCOMPRESS=(
+	"mv ../make-${VERSION}/.??* ."
+	"mv ../make-${VERSION}/* ."
+	"rmdir ../make-${VERSION}"
+)
+
+# Modification from OpenOrienteering / mingw-builds-3.0.0
 PATCHES=(
 	make/make-linebuf-mingw.patch
 	make/make-getopt.patch
-	make/make-Windows-Add-move-to-sh_cmds_dos.patch
-	make/make-git-fix-config-files.patch
 )
 
 #
 
+# Modification from OpenOrienteering / mingw-builds-3.0.0
 EXECUTE_AFTER_PATCH=(
-	"cp -rf $PATCHES_DIR/make/doc/* $SRCS_DIR/make_git/doc/"
-	"cp -rf $PATCHES_DIR/make/po/* $SRCS_DIR/make_git/po/"
 	"autoreconf -i"
 )
 
@@ -75,8 +82,11 @@
 
 #
 
+# Modification from OpenOrienteering / mingw-builds-3.0.0
 MAKE_FLAGS=(
 	-j$JOBS
+	do-po-update
+	scm-update
 	all
 )
 
@@ -87,4 +97,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-api.sh mingw-builds/scripts/mingw-w64-api.sh
--- mingw-builds.orig/scripts/mingw-w64-api.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-api.sh	2013-08-19 03:13:05 +0000
@@ -44,7 +44,7 @@
 SRC_DIR_NAME=mingw-w64-headers
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-headers
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-headers
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=runtime
 
 #
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-crt.sh mingw-builds/scripts/mingw-w64-crt.sh
--- mingw-builds.orig/scripts/mingw-w64-crt.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-crt.sh	2013-08-19 03:13:05 +0000
@@ -44,7 +44,7 @@
 SRC_DIR_NAME=mingw-w64-crt
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-crt
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-crt
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=runtime
 
 #
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-libraries-libmangle.sh mingw-builds/scripts/mingw-w64-libraries-libmangle.sh
--- mingw-builds.orig/scripts/mingw-w64-libraries-libmangle.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-libraries-libmangle.sh	2013-08-19 03:13:05 +0000
@@ -39,7 +39,7 @@
 SRC_DIR_NAME=libmangle
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-libraries/libmangle
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-libraries/libmangle
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=extra
 
 #
@@ -81,4 +81,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-tools-gendef.sh mingw-builds/scripts/mingw-w64-tools-gendef.sh
--- mingw-builds.orig/scripts/mingw-w64-tools-gendef.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-tools-gendef.sh	2013-08-19 03:13:05 +0000
@@ -39,7 +39,7 @@
 SRC_DIR_NAME=gendef
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-tools/gendef
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-tools/gendef
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=extra
 
 #
@@ -82,4 +82,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-tools-genidl.sh mingw-builds/scripts/mingw-w64-tools-genidl.sh
--- mingw-builds.orig/scripts/mingw-w64-tools-genidl.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-tools-genidl.sh	2013-08-19 03:13:05 +0000
@@ -39,7 +39,7 @@
 SRC_DIR_NAME=genidl
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-tools/genidl
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-tools/genidl
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=extra
 
 #
@@ -81,4 +81,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-tools-genpeimg.sh mingw-builds/scripts/mingw-w64-tools-genpeimg.sh
--- mingw-builds.orig/scripts/mingw-w64-tools-genpeimg.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-tools-genpeimg.sh	2013-08-19 03:13:05 +0000
@@ -39,7 +39,7 @@
 SRC_DIR_NAME=genpeimg
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-tools/genpeimg
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-tools/genpeimg
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=extra
 
 #
@@ -81,4 +81,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/mingw-w64-tools-widl.sh mingw-builds/scripts/mingw-w64-tools-widl.sh
--- mingw-builds.orig/scripts/mingw-w64-tools-widl.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mingw-w64-tools-widl.sh	2013-08-19 03:13:05 +0000
@@ -39,7 +39,7 @@
 SRC_DIR_NAME=widl
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-tools/widl
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-tools/widl
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=extra
 
 #
diff -urwN -x toolchains mingw-builds.orig/scripts/mpc.sh mingw-builds/scripts/mpc.sh
--- mingw-builds.orig/scripts/mpc.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mpc.sh	2013-08-19 03:13:05 +0000
@@ -80,4 +80,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/mpfr.sh mingw-builds/scripts/mpfr.sh
--- mingw-builds.orig/scripts/mpfr.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/mpfr.sh	2013-08-19 03:13:05 +0000
@@ -80,4 +80,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/ncurses.sh mingw-builds/scripts/ncurses.sh
--- mingw-builds.orig/scripts/ncurses.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/ncurses.sh	2013-08-19 03:13:05 +0000
@@ -92,4 +92,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp README $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/ppl.sh mingw-builds/scripts/ppl.sh
--- mingw-builds.orig/scripts/ppl.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/ppl.sh	2013-08-19 03:13:05 +0000
@@ -86,4 +86,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/python2.sh mingw-builds/scripts/python2.sh
--- mingw-builds.orig/scripts/python2.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/python2.sh	2013-08-19 03:13:05 +0000
@@ -147,4 +147,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp LICENSE* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/readline.sh mingw-builds/scripts/readline.sh
--- mingw-builds.orig/scripts/readline.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/readline.sh	2013-08-19 03:13:05 +0000
@@ -82,4 +82,10 @@
 	install
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/winpthreads-x32.sh mingw-builds/scripts/winpthreads-x32.sh
--- mingw-builds.orig/scripts/winpthreads-x32.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/winpthreads-x32.sh	2013-08-19 03:13:05 +0000
@@ -39,7 +39,7 @@
 SRC_DIR_NAME=winpthreads
-URL=http://mingw-w64.svn.sourceforge.net/svnroot/mingw-w64/trunk/mingw-w64-libraries/winpthreads
+URL=http://svn.code.sf.net/p/mingw-w64/code/trunk/mingw-w64-libraries/winpthreads
 TYPE=svn
-REV=
+REV=5796 # Modification from OpenOrienteering
 PRIORITY=runtime
 
 #
@@ -77,4 +77,10 @@
 	$( [[ $STRIP_ON_INSTALL == yes ]] && echo install-strip || echo install )
 )
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp COPYING* $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
diff -urwN -x toolchains mingw-builds.orig/scripts/zlib.sh mingw-builds/scripts/zlib.sh
--- mingw-builds.orig/scripts/zlib.sh	2013-05-22 09:52:48 +0000
+++ mingw-builds/scripts/zlib.sh	2013-08-19 03:13:05 +0000
@@ -38,8 +38,8 @@
 VERSION=1.2.7
 NAME=zlib-${VERSION}
 SRC_DIR_NAME=zlib-${VERSION}
-URL=http://zlib.net/zlib-${VERSION}.tar.bz2
-TYPE=.tar.bz2
+URL=http://sourceforge.net/projects/libpng/files/zlib/${VERSION}/zlib-${VERSION}.tar.gz # Modification from OpenOrienteering
+TYPE=.tar.gz # Modification from OpenOrienteering
 PRIORITY=prereq
 
 #
@@ -61,4 +61,10 @@
 
 INSTALL_FLAGS=()
 
+# Modification from OpenOrienteering
+EXECUTE_AFTER_INSTALL=(
+	"mkdir -p $PREFIX/licenses/$SRC_DIR_NAME"
+	"cp README $PREFIX/licenses/$SRC_DIR_NAME/"
+)
+
 # **************************************************************************
