--- qt-everywhere-opensource-src-4.8.1/src/corelib/codecs/codecs.pri.orig	2012-07-26 11:55:00.000000000 +0200
+++ qt-everywhere-opensource-src-4.8.1/src/corelib/codecs/codecs.pri	2012-07-26 11:55:15.000000000 +0200
@@ -56,5 +56,24 @@
                         ../plugins/codecs/tw/qbig5codec.cpp \
                         ../plugins/codecs/jp/qfontjpcodec.cpp
         }
+} else {
+                # no iconv, so we put all plugins in the library
+                HEADERS += \
+                        ../plugins/codecs/cn/qgb18030codec.h \
+                        ../plugins/codecs/jp/qeucjpcodec.h \
+                        ../plugins/codecs/jp/qjiscodec.h \
+                        ../plugins/codecs/jp/qsjiscodec.h \ 
+                        ../plugins/codecs/kr/qeuckrcodec.h \
+                        ../plugins/codecs/tw/qbig5codec.h \
+                        ../plugins/codecs/jp/qfontjpcodec.h
+                SOURCES += \
+                        ../plugins/codecs/cn/qgb18030codec.cpp \
+                        ../plugins/codecs/jp/qjpunicode.cpp \
+                        ../plugins/codecs/jp/qeucjpcodec.cpp \
+                        ../plugins/codecs/jp/qjiscodec.cpp \
+                        ../plugins/codecs/jp/qsjiscodec.cpp \ 
+                        ../plugins/codecs/kr/qeuckrcodec.cpp \
+                        ../plugins/codecs/tw/qbig5codec.cpp \
+                        ../plugins/codecs/jp/qfontjpcodec.cpp
 }
 symbian:LIBS += -lcharconv
