diff -up Proj/configure.orig Proj/configure
--- Proj/configure.orig	2015-11-17 09:19:22.915735180 +0100
+++ Proj/configure	2015-11-17 09:25:42.834099078 +0100
@@ -11317,6 +11317,25 @@ linux*oldld* | linux*aout* | linux*coff*
   dynamic_linker=no
   ;;
 
+# From libtool commit 8eeeb00daef8c4f720c9b79a0cdb89225d9909b6
+linux*android*)
+  version_type=none # Android doesn't support versioned libraries.
+  need_lib_prefix=no
+  need_version=no
+  library_names_spec='$libname$release$shared_ext'
+  soname_spec='$libname$release$shared_ext'
+  finish_cmds=
+  shlibpath_var=LD_LIBRARY_PATH
+  shlibpath_overrides_runpath=yes
+
+  # This implies no fast_install, which is unacceptable.
+  # Some rework will be needed to allow for fast_install
+  # before this can be enabled.
+  hardcode_into_libs=yes
+
+  dynamic_linker='Android linker'
+  ;;
+
 # This must be glibc/ELF.
 linux* | k*bsd*-gnu | kopensolaris*-gnu | gnu*)
   version_type=linux # correct to gnu/linux during the next big refactor
